processor/src/chiplets/aux_trace/mod.rs

```diff
@@ -2,47 +2,62 @@ use alloc::vec::Vec;
 
 use miden_air::trace::main_trace::MainTrace;
 use vm_core::Kernel;
+use wiring_bus::WiringBusBuilder;
 
-use super::{super::trace::AuxColumnBuilder, Felt, FieldElement};
+use super::{super::trace::AuxColumnBuilder, Felt, FieldElement, ace::AceHints};
 
 mod bus;
-pub use bus::BusColumnBuilder;
+pub use bus::{
+    BusColumnBuilder, build_ace_memory_read_element_request, build_ace_memory_read_word_request,
+};
 
 mod virtual_table;
 pub use virtual_table::ChipletsVTableColBuilder;
 
+mod wiring_bus;
+
 /// Constructs the execution trace for chiplets-related auxiliary columns (used in multiset checks).
 #[derive(Debug)]
 pub struct AuxTraceBuilder {
     kernel: Kernel,
+    ace_hints: AceHints,
 }
 
 impl AuxTraceBuilder {
     // CONSTRUCTORS
     // --------------------------------------------------------------------------------------------
 
-    pub fn new(kernel: Kernel) -> Self {
-        Self { kernel }
+    pub fn new(kernel: Kernel, ace_hints: AceHints) -> Self {
+        Self { kernel, ace_hints }
     }
 
     // COLUMN TRACE CONSTRUCTOR
     // --------------------------------------------------------------------------------------------
 
-    /// Builds and returns the Chiplets's auxiliary trace columns. Currently this consists of
-    /// a single bus column `b_chip` describing chiplet lookups requested by the stack and
-    /// provided by chiplets in the Chiplets module.
+    /// Builds and returns the Chiplets's auxiliary trace columns. This consists of:
+    ///
+    /// 1. a bus column `b_chip` describing requests made by the stack and decoder and responses
+    ///    received from the chiplets in the Chiplets module,
+    /// 2. a column acting as both virtual tables, one for the sibling table used by the hasher
+    ///    chiplet and the other for the kernel procedure table, and as a bus between the memory
+    ///    chiplet and the ACE chiplet.
```

Comment:
note that the virtual table (bus) column is now expanded to handle the messages between ACE and memory. We should probably include this in the updated docs.